src/main/java/uk/nhs/hee/tis/revalidation/connection/service/MasterElasticSearchService.java

```diff
@@ -74,73 +60,34 @@
 
   /**
    * find all trainee from ES Master Index.
-   * (with Scroll API to avoid ES index max_result_window excess error)
+   * (by `scroll` to avoid ES index max_result_window excess error)
    */
-  @SneakyThrows
-  public List<ConnectionInfoDto> findAllScroll() {
-
-    List<ConnectionInfoDto> results = new ArrayList<>();
-
-    RestHighLevelClient client = new RestHighLevelClient(
-        RestClient.builder(
-            new HttpHost(ES_URL.getHost(), ES_URL.getPort(), ES_URL.getProtocol())));
-
-    try {
-
-      // Initial search
-      SearchRequest searchRequest = new SearchRequest("masterdoctorindex");
-      SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
-      searchSourceBuilder.size(PAGE_SIZE);
-      searchRequest.source(searchSourceBuilder);
-      searchRequest.scroll("5m");
-      SearchResponse searchResponse = client.search(searchRequest, RequestOptions.DEFAULT);
-      String scrollId = searchResponse.getScrollId();
-      SearchHits hits = searchResponse.getHits();
-      // add records to results
-      results.addAll(hitsToDto(hits));
+  public List<ConnectionInfoDto> findAllBySpringDataScroll() {
+    final ObjectMapper mapper = new ObjectMapper();
+    List<MasterDoctorView> masterViews = new ArrayList<>();
+    List<String> scrollIdList = new ArrayList<>();
```

Comment:
Maybe rename to `scrollIds` to stay consistent with the naming style used for `masterViews`

Reply:
It is renamed to `scrollIds`. Thanks Andy.